pages/index.js

```diff
@@ -1,3 +1,10 @@
+import Navbar from "../components/Navbar/Navbar";
+
 export default function Home() {
-  return <h1>Willkommen zum 🐙-Capstone-Template!</h1>;
+  return (
+    <>
+      <h1>Willkommen zum 🐙-Capstone-Template!</h1>
+      <Navbar />
+    </>
```

Comment:
You should put your `Navbar` in your `_app.js` instead to ensure that it will be rendered on every page and not only on your homepage.